cfme/v2v/migrations.py

```diff
@@ -712,14 +726,16 @@ class MigrationPlanCollection(BaseCollection):
     ENTITY = MigrationPlan
 
     def create(self, name, infra_map, vm_list, description=None, csv_import=False,
-               start_migration=False):
+               pre_playbook=False, post_playbook=False, start_migration=False):
```

Comment:
I think `None` is a more appropriate default here.

Reply:
uhmm..let me think..so yes we are just checking in condition so `None` also will be good in this condition